src/redisearch_rs/inverted_index/src/freqs_only/tests.rs

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+use std::io::Cursor;
+
+use crate::{Decoder, DecoderResult, Delta, Encoder, RSIndexResult, freqs_only::FreqsOnly};
+
```

Comment:
We should test the unhappy case: reader/writer failures, delta bigger than u32::MAX.